flow360/component/simulation/translator/solver_translator.py

```diff
@@ -730,8 +732,7 @@ def get_solver_json(
             if isinstance(op.thermal_state.material.dynamic_viscosity, Sutherland)
             else -1
         ),
-        # pylint: disable=protected-access
-        "muRef": op.thermal_state._mu_ref(mesh_unit),
+        "muRef": op.thermal_state.dynamic_viscosity.v.item(),
```

Comment:
This will actually return the nondimensional dynamic_viscosity even when it's not a field? @benflexcompute and I spent so much time looking for a good solution when we weren't aware that this would work.

Reply:
yes, it will. It will call either Sutherlands reference viscosity or Viscosity directly. The units are converted to flow360 in the preprocess stage making sure here they will be flow360_viscosity_units